src/main/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueActor.scala

```diff
@@ -130,6 +138,18 @@ private[impl] class LaunchQueueActor(
     suspendedLaunchersMessages += actorRef -> deferredMessages
   }
 
+  private[this] def receiveTaskUpdate: Receive = {
+    case update: TaskStatusUpdate =>
+      import context.dispatcher
+      launchers.get(update.appId) match {
+        case Some(actorRef) =>
+          val eventualCount: Future[QueuedTaskCount] =
```

Comment:
It is something that intellij suggests for futures. I can change it if you dislike it.